packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -267,6 +280,30 @@ describe('Sayt Component', () => {
     });
   });
 
+  describe('debounceSaytRequests', () => {
+    it('should debounce the requestSaytAutocompleteTerms and requestSaytProducts methods', () => {
+      const delay = sayt.debounceTime;
+      const debounceSettings = { 'leading': true, 'trailing': true };
+      const debounceStub = stub(Lodash, 'debounce');
+      const debounceSaytRequests = stub(sayt, 'debounceSaytRequests');
+
+      const requestSaytAutocompleteTerms = debounceStub(sayt.requestSaytAutocompleteTerms(), delay, debounceSettings);
+      const requestSaytProducts = debounceStub(sayt.requestSaytProducts(), delay, debounceSettings);
+      const autocompleteTermsReturn = debounceSaytRequests.returns(requestSaytAutocompleteTerms);
+      const productsReturn = debounceSaytRequests.returns(requestSaytProducts);
+
+      sayt.debounceSaytRequests(sayt.requestSaytAutocompleteTerms);
+      sayt.debounceSaytRequests(sayt.requestSaytProducts);
+
+      expect(debounceStub).to.be.calledTwice;
+      expect(debounceSaytRequests).to.be.calledTwice;
```

Comment:
We've been avoiding asserting exactly how many times a method has been called, and have instead been relying on asserting (as you have done below) that the right arguments are passed to methods.